LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2021 Quarkiverse.io
```

Comment:
No sure Quarkiverse.io is something that can own a coryright.
Even the Quarkus license file shows `Copyright [yyyy] [name of copyright owner]`.
I'm not a license expert so I don't have the answer but maybe this

Reply:
You can just copy the apache license contents as is from https://www.apache.org/licenses/LICENSE-2.0.txt

Reply:
It's the same ;)

Reply:
Reverted to what comes generated.